/home/jupyter/.cache/wandb/logs/core-debug-20250428_221236.log